### DISCUSSION TITLE
Virtual IP Manager doesn’t work in Google Cloud

### DISCUSSION THREAD

**Discussion by 0nurulusoy** [autobase-tech/autobase#321] [General]
Hey, when i enable vip, it assign an virtual address to one of the interfaces but somehow GCP doesn't use traditional LAN mechanisms, when you try to access an instance local ip between instances, it resolves an dns, since it doesn't crate dns for vip, vip manager doesn't work on GCP. Any ideas about how to resolve this issue.

**Comment by vitabaks** [comment]
I have not yet had the task of deploying a cluster with VIP in GCP, so I am not ready to give specific recommendations.  It is important that the cloud provider allows you to use an additional floating ip address that automatically switches to another server in the database cluster in case the Primary server failover. If you choose a scheme using VIP.

**Reply by vitabaks** [reply]
as for dns, for example, I never use it.  Unless in the scheme with consul where a dns record is provided as an entry point to the database by means of consul service discovery.
